lib/jekyll/commands/watch.rb

```diff
@@ -11,17 +11,7 @@ def init_with_program(prog); end
       # Continuously watch if `watch` is set to true in the config.
       def process(options)
         Jekyll.logger.log_level = :error if options["quiet"]
-        watch(site, options) if options["watch"]
-      end
-
-      # Watch for file changes and rebuild the site.
-      #
-      # site - A Jekyll::Site instance
-      # options - A Hash of options passed to the command
-      #
-      # Returns nothing.
-      def watch(_site, options)
-        Jekyll::Watcher.watch(options)
+        Jekyll::Watcher.watch(options) if options["watch"]
```

Comment:
Since `_site` wasn't used anyway, this method was doing nothing more than calling `Jekyll::Watcher.watch`, so let's just call it directly.